docs/guides/custom-images.rst

```diff
@@ -0,0 +1,68 @@
+Using Custom Docker Images on Compute Engine
+=====================
+
+When running workflows remotely on Compute Engine, your tasks run in a container that uses one of the
+``orquestra-sdk-base`` images depending on whether you have requested a GPU or not (you can see their source code
+`here <https://github.com/zapatacomputing/orquestra-workflow-sdk/blob/main/docker/Dockerfile>`_ and
+`here <https://github.com/zapatacomputing/orquestra-workflow-sdk/blob/main/docker/cuda.Dockerfile>`_).
+If your task needs extra dependencies that you cannot install using imports (such as native libraries or
+executable binaries), you can use one of the above images as your base image to publish a new image that you
+will use as your container when running your task.
+
+Building an Image
+-----------------
+
+The base images are hosted at Orquestra Nexus. There are two versions of the base image per each SDK version, one with
+CUDA dependencies installed and one without. Their names follow the pattern
+``hub.nexus.orquestra.io/zapatacomputing/orquestra-sdk-base:<SDK version>[-cuda]``. For example,
+``hub.nexus.orquestra.io/zapatacomputing/orquestra-sdk-base:0.48.0`` or
+``hub.nexus.orquestra.io/zapatacomputing/orquestra-sdk-base:0.49.0-cuda``. You should pick the one that matches
+your SDK version and your GPU needs (``-cuda`` needs to be picked for GPU workflows).
+
+The base images run as a user named ``orquestra`` with uid ``1000``. Since these are Ubuntu based images, you
+can install any package that you need from Ubuntu repositories (by doing ``RUN apt install <package name>``).
+However, before doing so, you need to temporarily switch to the root user (via ``USER root`` directive) and
+restore back to user ``orquestra`` once you're done (``USER orquestra``).
+
+.. note::
+
+    You can use `standard OCI annotations <https://github.com/opencontainers/image-spec/blob/main/annotations.md>`_ to add metadata to your images.
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏Can you add a callout like this to say that it is the user's responsibility to keep their images up to date with the new SDK versions?